Configuration/StandardSequences/python/DQMSaverAtJobEnd_cff.py

```diff
@@ -15,6 +15,11 @@
 
 dqmSaver.saveByRun = -1
 dqmSaver.saveAtJobEnd = True  
-dqmSaver.forceRunNumber = 1
+dqmSaver.forceRunNumber = 999999
```

Comment:
@schneiml could you please explain the reason for this and its impacts as it is a global change, not just for the new workflow? Is this basically a dummy parameter affecting just the output file name in https://cmssdt.cern.ch/lxr/source/DQMServices/Components/src/DQMFileSaver.cc#0097
and the internal location of saving the plot in dbe:save?

Reply:
@fabiocos It turns out it is used, here: https://github.com/cms-sw/cmssw/blob/5e0d560e2bb7ddd92fe3f1d086e08f4d9b07f910/Configuration/Applications/python/ConfigBuilder.py#L1971-L1972

I was not aware of that, and it breaks the pre2 relval now.

Just removing the  `--harvesting AtJobEnd` option in wf 503 et.al. seems to not work either, then we get run number 0. 

See also #25272 , FYI @prebello @zhenhu

Reply:
Only the GEN workflows use the `--harvesting AtEndJob` option. On the conceptual level, I don't see why they do that, since there should only be one run there.

In practise, I checked what happens if I make them use `--harvesting AtEndRun` as most of the other WFs do, but this does not work, since apparently the internal CMSSW run number in these jobs is 0, and the `AtEndJob` was required to force it to 1. 

I think the proper solution would be to modify some part of the configuration in some way (many ways are possible) to set `dqmSaver.forceRunNumber = 1` for the GEN jobs, and remove the `--harvesting AtEndJob`. Note that this might change the behaviour of the DQM in these jobs, but probably for the better.

Reply:
@prebello  Sang Hyun Ko  reported on JIRA that it works for him to just remove the `--harvesting AtEndJob`. Not sure what happend in my case, I can't reproduce the run 0 file now...

So, we should just remove the `--harvesting AtEndJob` (https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2320) and everything should be good.